src/main.rs

```diff
@@ -204,11 +199,12 @@ fn translate(geth_spec: GethSpec) -> ParitySpec {
 	let start_nonce = ask_start_nonce();
 
 	let parity_params = ParityParams {
+		registrar: "0x81a4b044831c4f12ba601adb9274516939e9b8a2".into(),
```

Comment:
Registrar should be queried for and only default to mainnet address if none is given, as this hardcoded value is not reasonable for some general reason.